src/equations/acoustic_perturbation_2d.jl

```diff
@@ -112,9 +112,9 @@ A constant initial condition where the state variables are zero and the mean flo
 Uses the global mean values from `equations`.
 """
 function initial_condition_constant(x, t, equations::AcousticPerturbationEquations2D)
-    v1_prime = 0.0
-    v2_prime = 0.0
-    p_prime_scaled = 0.0
+    v1_prime = 0
+    v2_prime = 0
+    p_prime_scaled = 0
```

Comment:
This relies on implicit conversion to the correct floating point type in `SVector`. I am wondering what our "correct" approach here should be (reliance on implicit conversion or using an explicit `RealT`). What was your conclusion on this @ranocha @huiyuxie?

Reply:
IIRC, it was "let the authors decide".

Reply:
Fair enough. Maybe this should be documented in the conventions section as well then.